README.md

```diff
@@ -39,6 +39,18 @@ installed: /etc/systemd/system/v2ray@.service
 # bash <(curl -L https://raw.githubusercontent.com/v2fly/fhs-install-v2ray/master/install-release.sh)
 ```
 
+### V2Ray 組態
+1. 產生配置文檔 [https://intmainreturn0.com/v2ray-config-gen](https://intmainreturn0.com/v2ray-config-gen/)
```

Comment:
This config generation is wrong, you should not add it

Reply:
As an alternative, I could include a template config.json.template file with instructions/script on how to generate a secure UUID. Would this be better and acceptable?